components/brave_rewards/resources/ui/components/tableAdsHistory/index.tsx

```diff
@@ -133,7 +133,7 @@ export default class TableAdsHistory extends React.PureComponent<Props, {}> {
     return (
       <StyledCategoryContentDiv>
         <StyledCategoryName>
-          {content.category}
+          {content.category === 'untargeted' ? 'general' : content.category}
```

Comment:
Categories in Ads History are not localized as they are downloaded with the catalog at this time, which is the reason `general` has not been localized for consistency